src/node_api.h

```diff
@@ -116,21 +117,18 @@ NAPI_EXTERN napi_status napi_create_array_with_length(napi_env env,
                                                       size_t length,
                                                       napi_value* result);
 NAPI_EXTERN napi_status napi_create_number(napi_env env,
-                                           double val,
+                                           double value,
                                            napi_value* result);
 NAPI_EXTERN napi_status napi_create_string_utf8(napi_env env,
-                                                const char* s,
-                                                size_t length,
+                                                const char* buf,
+                                                size_t bufsize,
```

Comment:
Yes, was trying to be consistent but wasn't entirely satisfied with ```buf``` and ```bufsize```, either. ```bufsize``` seems to indicate byte count when it should be length in characters. I'll rename these ```str``` and ```length``` to clear up confusion.